### DISCUSSION TITLE
Is it possible to bind motion sensors or door switches?

### DISCUSSION THREAD

**Discussion by stoinov** [Koenkk/zigbee2mqtt#6935] [Devices]
I am trying to bind [MCCGQ11LM](https://www.zigbee2mqtt.io/devices/MCCGQ11LM.html) and [RTCGQ11LM](https://www.zigbee2mqtt.io/devices/RTCGQ11LM.html) to a [GL-MC-001](https://www.zigbee2mqtt.io/devices/GL-MC-001.html) but I am getting:
`Nothing to bind from 'friendly_name' to 'gle01-light'`
I tried it with an empty group too but the same behavior is shown.

Is this a general limitation for such sensors, or are those two Aqara models not capable of binding?

I can see that someone else is trying with IKEA motion sensor [here](https://github.com/Koenkk/zigbee2mqtt/issues/6438).

**Comment by Koenkk** [comment]
In order for this to be supported, the motion sensor should support the `genOnOff` cluster. The MCCGQ11LM and RTCGQ11LM don't do this. The only device I've seen to do this is the IKEA motion sensor.

**Reply by stoinov** [reply]
Thanks - I searched in the device database and saw that only handful of remotes and sensor supports it.

Does that mean that all lights and switches supports being receiver of the binding command, or is this also a hardware limitation that I need to be aware of?

**Reply by Koenkk** [reply]
It will work for all devices which use standard Zigbee commands to turn on/off (almost everything except some specific TuYa devices)

**Reply by KidA001** [reply]
@Koenkk I have Tradfri motion sensor which supports genOnOff but when bound to a zigbee light switch it does not turn it on. I've posted details here: https://github.com/Koenkk/zigbee2mqtt/discussions/18855
